install.sh

```diff
@@ -7,6 +7,12 @@ if [[ -n "$MSYSTEM" ]]; then
   exit 1
 fi
 
+# Check if script was executed with root/sudo
+if [ `id -u` -ne 0 ]
+  then echo Please run this script as root or using sudo!
+  exit
+fi
+
```

Comment:
You should add another condition if there is no `CI=true` environment variable, then this check would be valid.

Reply:
do you want to have it ac environment variable or as additional command parameter?

Reply:
`CI=true` should be from environment variable. It's for the CI to be able to pass.

Reply:
okay